src/sparseml/pytorch/torchvision/train.py

```diff
@@ -466,6 +466,25 @@ def collate_fn(batch):
         return
 
     manager = ScheduledModifierManager.from_yaml(args.recipe_path)
+    if args.one_shot:
+        manager.apply(model)
+        checkpoint = {
+            "model": model_without_ddp.state_dict(),
```

Comment:
need to save the (composed) recipe for model state reproduction on re-load

Reply:
Good call. Will wait for checkpoint to be merged in. Closing this one for now to de-clutter